lms/views/basic_lti_launch.py

```diff
@@ -188,7 +194,64 @@ def url_configured_basic_lti_launch(self):
         LMS, which passes it back to us in each launch request. All we have to
         do is pass the URL to Via.
         """
-        return self.basic_lti_launch(self.request.parsed_params["url"])
+        groups = None
+
+        lti_user = self.request.lti_user
+        canvas = self.request.find_service(name="canvas_api_client")
+        if True:  # "is_group_assigment":
```

Comment:
Not very nice code ahead but it just gets the courses a student belongs to and finds the one in the group set. Syncs to H and sets `groups` to make it available in the client.